database/scripts/0002-populate-data.sql

```diff
@@ -19,4 +19,9 @@ INSERT INTO public.event_rating(would_recommend, created_by, event_id) VALUES
     (true, 2, 1),
     (false, 1, 2),
     (false, 2, 2),
-    (true, 2, 5);
\ No newline at end of file
+    (true, 2, 5);
+
+-- Postgres requires us to reset autoincrement counters manually if we're inserting data by explicitly setting IDs
```

Comment:
@jurebb This fixes those intermittent "Validation error" issues:

https://til.codes/dont-forget-to-update-the-sequence-in-postgresql-after-a-copy-command/

Reply:
Awesome, great find :D